js/src/components/ContentNavigation.js

```diff
@@ -50,14 +57,11 @@ const ContentNavigation = ({ isIntro, sections, currentLesson, nextLesson, curre
 
             {noNextLesson && !nextIsLesson && !nextIsQuiz && (
               <Button {...buttonProps} href={`/node/${currentLesson.id}/finish`}>
-                {disabled ?
-                  completeAnswer :
-                  (
-                    currentLesson.finishButtonText === '' ?
-                    Drupal.t('Finish', {}, { context: 'ANU LMS' }) :
-                    currentLesson.finishButtonText
-                  )
-                }
+                {disabled
+                  ? completeAnswer
+                  : currentLesson.finishButtonText === ''
+                  ? Drupal.t('Finish', {}, { context: 'ANU LMS' })
+                  : currentLesson.finishButtonText}
```

Comment:
a nested ternary is kinda hard to read. can you extract it into a function (or multiple)?